src/App.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { BrowserRouter as Router, Route, Routes } from 'react-router-dom';
+import SignUp from './pages/SignUp';
+import LogIn from './pages/LogIn';
+import WelcomePage from './pages/WelcomePage';
+import Loader from './components/loader/Loader';
+import Layout from './components/Layout';
+import AddReservations from './components/AddReservations';
+import Cars from './pages/Cars';
+import Reservations from './components/Reservations';
+import ProtectedRoutes from './components/ProtectedRoutes';
+import CarDetails from './components/CarDetails';
+import AddCar from './pages/AddCar';
+import DeleteCar from './pages/DeleteCar';
+
+function App() {
```

Comment:
- [ ]  Please, kindly refactor this and all other functions in your project to ES6 arrow functions, so your code complies with the latest standards.

Refactoring is easy:

```javascript
const App = () => {
  // ...
}
```

## Explanation

The `function` keyword has been soft deprecated in almost all the software industry in favor of ES6 arrow functions. This means, almost nobody uses it anymore, so much that it's considered a non-best practice.

You can do a search in Vscode for the "function" word, and correct as needed.  Shouldn't take you more than 5-10 min so don't worry:



![image](https://github.com/tumainimaganiko/front-end-capstone-group/assets/57421823/728e3f4d-2c10-4939-989b-1715dc0a7de6)

That way, your code will meet the latest standards. 💪

Reply:
@Meltrust  Thanks for pointing out this we are happy to receive this suggestion and I'm happy to inform you we have already refactored all our codes